ports/bsa/portfile.cmake

```
@@ -0,0 +1,24 @@
+vcpkg_fail_port_install(ON_TARGET "Linux" "OSX" ON_ARCH "x86")

```

```suggestion
vcpkg_fail_port_install(ON_TARGET "Linux" "OSX" "UWP" ON_ARCH "x86")
```

```
     4>D:\buildtrees\bsa\src\2.0.3-80256db965.clean\src\bsa\detail\common.cpp(228,20): error C2039: 'CreateFileW': is not a member of '`global namespace'' [D:\buildtrees\bsa\x64-uwp-dbg\src\bsa.vcxproj]
     4>D:\buildtrees\bsa\src\2.0.3-80256db965.clean\src\bsa\detail\common.cpp(228,31): error C3861: 'CreateFileW': identifier not found [D:\buildtrees\bsa\x64-uwp-dbg\src\bsa.vcxproj]
```
`CreateFileW` does not support uwp.